video_to_image/requirements.txt

```
@@ -0,0 +1,2 @@
+numpy==1.26.0
+opencv-python==4.8.0.76

```

In order for the [top-level instructions](https://github.com/centaurlabs/centaur-recipes#installing-requirements-and-running-a-notebook) to work, we need `jupyter` and `ipykernel` installed as well.
```suggestion
jupyter
ipykernel
numpy==1.26.0
opencv-python==4.8.0.76

```